Libraries.txt

```diff
@@ -0,0 +1,915 @@
+Open Source libraries used in the Scanbot Barcode Scanner SDK for Cordova version 3.0.1:
+
+------------------------------------------------------------------------------------------------------------------------
+
+Open Source Libraries used in the Scanbot Barcode Scanner SDK for Android Version 3.0.1:
```

Comment:
Should be full sdk instead of barcode sdk

Reply:
Fixed